src/OrderTransfer/order_transfer.go

```diff
@@ -0,0 +1,55 @@
+package ordertransfer
```

Comment:
place OrderTransfer and OrderWebAdapter in their own application folder so that it follows the scheme we have for the rest of the modules